### PR TITLE
Layout tutorial updates

### PR DESCRIPTION
**Pull request by mbektasbbg**
- Moves higher lever containers into main layout notebook
- removes widget metadata from higher level containers notebook for size reduce and speed up
- simplifications
- new examples
- clean up

### CODE REVIEW

**Comment by martinRenou** [comment]
Sorry for the conflicts @mbektasbbg, there should be only conflicts on the naming of the Notebooks (not in the content). Those Notebooks are now `05.0x` instead of `06.0x`.

File: notebooks/06.01-Widget_Layout.ipynb [hunk 1]
```diff
@@ -586,19 +857,604 @@
...
+    "                                 fig_margin=dict(top=0, bottom=0, left=0, right=0))\n",
+    "        else:\n",
+    "            sc_x = scales_x[j]\n",
+    "            sc_y = bq.LinearScale()\n",
+    "        \n",
+    "            hist = bq.Hist(sample=data[:,i], scales={'sample': sc_x, 'count': sc_y})\n",
+    "            \n",
+    "            gs[i, j] = bq.Figure(marks=[hist], layout=Layout(width='auto', height='auto'),\n",
+    "                                 fig_margin=dict(top=0, bottom=0, left=0, right=0))\n",
+    "gs"
```
Comment (mwcraig) [inline notebooks/06.01-Widget_Layout.ipynb:1434]: I got the image below when I ran this cell -- just wanted to confirm this is as expected.

JupyterLab

Comment (mwcraig) [reply notebooks/06.01-Widget_Layout.ipynb:1434]: Ah, I see this was copied over from 06.05; had forgotten what it looks like.

Comment (mbektasbbg) [reply notebooks/06.01-Widget_Layout.ipynb:1434]: yes this is a grid of scatter plots and bar charts. I will try to make scatter plot look nicer.

we can remove 6.05 now. I wont cover 6.07 (widget label styling), but will ask attendees to go over themselves. I already mention most of the items in 6.09 (widget/button styling) in the main notebook. We can keep or remove, not sure. I won't cover 7.00 (OPTIONAL widget layout exercises). I am not too sure what optional is meant here. Should we make 6.07 optional as well?

**Comment by mbektasbbg** [comment]
> Sorry for the conflicts @mbektasbbg, there should be only conflicts on the naming of the Notebooks (not in the content). `06.05` became `06.02` (it was previously jumping from `01` to `05`).

no worries @martinRenou

**Comment by mwcraig** [comment]
> I am not too sure what optional is meant here. Should we make 6.07 optional as well?

The ones with OPTIONAL are the ones we are not planning to cover.

I'm watching a tutorial right now...this year it definitely the case that less is better!

**Comment by mbektasbbg** [comment]
OK @mwcraig makes sense. I will make changes for optional ones.

**Comment by mbektasbbg** [comment]
> Sorry for the conflicts @mbektasbbg, there should be only conflicts on the naming of the Notebooks (not in the content). `06.05` became `06.02` (it was previously jumping from `01` to `05`).

I just removed item labeled 7, once this PR is merged lets reorder.

**Comment by mwcraig** [comment]
Sounds good, merging

**Pull request merged**